### ISSUE TITLE
Can't run hte_intro since no complete cases

### ISSUE THREAD

**Issue by MaxGhenis** [markhwhiteii/blog#3]
Hi - I found your HTE intro really useful, so tried to recreate it, but ran into a hitch in the data loading step. Some fields like `canvass_trans_ratingsecond` seem to be always null, so the `complete.cases()` command leaves you with no data.

Here's my notebook, and see below. https://nbviewer.jupyter.org/github/MaxGhenis/random/blob/master/hte_intro.ipynb
![image](https://user-images.githubusercontent.com/6076111/56190751-c2ad6500-5fdf-11e9-87f7-ae311e23cc52.png)

**Comment by MaxGhenis** [comment]
Actually the issue is with the `respondent_t1` and `contacted` fields, which are all `NA` and filtered on.

**Comment by markhwhiteii** [comment]
sorry, just now seeing this—not sure why GitHub never told me about this. will look into it. I'd imagine something may have changed in different versions of packages.

**Comment by markhwhiteii** [comment]
Yeah, you're right it's based on that filtering. Not sure, but for some reason the file is being read-in differently than it was when I knitted the document in 2018. I might try a different function, such as `read.tsv`

**Comment by mwells21** [comment]
I ran into the same issue but was able to parse the .tab file using this:
`dat <- read.table("https://raw.githubusercontent.com/markhwhiteii/blog/master/hte_intro/broockman_kalla_replication_data.tab", header = T, sep = "\t", fill = TRUE)`

**Comment by markhwhiteii** [comment]
I'll take a look at this! I'm actually working on translating this to a formal paper so I haven't monitored this repo as much as I should. I'll take a look over the weekend. I assume something in `readr` changed. Thanks for posting your solution to it! I appreciate it.
